src/index.js

```diff
@@ -6,4 +6,9 @@ export default [
   'The leak is minor, we’ll just add more RAM later.',
   'Looks like it’s all working fine',
   'We’ll fix it later',
+  'If it works it works.',
```

Comment:
I believe the typical phrase here is "if it works, don't touch it"

Reply:
Ok will change to that